Documentation/compilation.rst

```diff
@@ -29,23 +28,23 @@ Add the following lines to your .bashrc (vi ~/.bashrc).
   export LIBRARY_PATH=$HOME/lib:$LIBRARY_PATH
   export LD_LIBRARY_PATH=$HOME/lib:$LD_LIBRARY_PATH
   export PKG_CONFIG_PATH=$HOME/lib/pkgconfig:$PKG_CONFIG_PATH
+  export CMAKE_PREFIX_PATH=$HOME
   export EDITOR=vi
   export CPATH=$HOME/include:$CPATH 
 
   # run "exec bash" or "source ~/.bashrc" to apply environment to the current shell
 
-Installing SCons
+Installing CMake
 ----------------
 
 .. code-block:: bash
 
-  wget http://prdownloads.sourceforge.net/scons/scons-3.0.5.tar.gz
-  tar -xaf scons-3.0.5.tar.gz
-  cd scons-3.0.5
-  python setup.py install --prefix=$HOME
-  cd ..
+  cd $(mktemp -d) && wget -qO- https://github.com/Kitware/CMake/releases/download/v3.10.2/cmake-3.10.2-Linux-x86_64.tar.gz | tar -xvz -C "." && mv "./cmake-3.10.2-Linux-x86_64" "${HOME}/bin/cmake"
```

Comment:
The whole command should be set in () brackets. This opens a subshell - the user remains in the same directory.